web/remote/nextcloud.go

```diff
@@ -226,7 +226,12 @@ func nextcloudUpstream(c echo.Context) error {
 		return jsonapi.BadRequest(errors.New("missing From parameter"))
 	}
 
-	if err := nc.Upstream(path, from); err != nil {
+	kind := nextcloud.MoveOperation
+	if c.QueryParam("Copy") != "" {
```

Comment:
Checking the actual content of the param would allow requesters to always pass it with values depending on the calling context.

Reply:
I don't really see what you would want to do

Reply:
Well, if I'm not mistaken, making a request with `Copy=false` as parameter would still be considered as a copy request.
My suggestion is to check the value of the parameter so only requests with `Copy=true` would make copies.

Reply:
Instead of `false`, you can use an empty string, if you really want to give `Copy=` in the query-string.

Reply:
Sure, but it's less obvious that's all. And makes calls using a variable a little bit more complex.
It's not a big deal but would make the API a bit nicer to use.